lib/cc/engine/analyzers/reporter.rb

```diff
@@ -93,8 +133,13 @@ def flay_options
           CCFlay.default_options.merge changes
         end
 
-        def debug(message)
-          $stderr.puts(message) if engine_config.debug?
+        require "thread"
```

Comment:
Oops. It was actually already required (tho not sure why?). I also moved the constant up for clarity.